common/src/IO/ObjSerializer.cpp

```diff
@@ -33,15 +37,28 @@ namespace TrenchBroom {
         texCoords(i_texCoords),
         normal(i_normal) {}
 
-        ObjFileSerializer::ObjFileSerializer(FILE* stream) :
-        m_stream(stream),
+        ObjFileSerializer::Face::Face(IndexedVertexList i_verts, String i_texture) :
+        verts(std::move(i_verts)),
+        texture(std::move(i_texture)) {}
+
+        ObjFileSerializer::ObjFileSerializer(const Path& path) :
+        m_objPath(path),
+        m_mtlPath(path.replaceExtension("mtl")),
+        m_objFile(std::make_unique<IO::OpenFile>(m_objPath, true)),
```

Comment:
Why on the heap? Because of copy / move semantics? I'd rather just make it non-copyable and non-moveable and use simple members.

Reply:
No reason just a leftover from how I wrote it.